store/displayPreferencesApi.ts

```diff
@@ -0,0 +1,216 @@
+import { DisplayPreferencesDto } from '@jellyfin/client-axios';
```

Comment:
As explained, this store is going to handle all the API-supported fields for DisplayPreferences and their sync state. I guessed that it would make more sense for new contributors to understand the difference between the stores that we're discussing in #801 and the ``settings`` one.